lib/active_job/queue_adapters/sidekiq_adapter_patch.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ActiveJob
+  module QueueAdapters
+    # Adds enqueued_jobs method consistent with the TestAdapter
+    #
+    class SidekiqAdapter
+      def enqueued_jobs
```

Comment:
Ah I wasn't aware this method wasn't available. Instead of monkey patching this, can we use available `Sidekiq` methods, I think this might work:
```
Sidekiq::Queue.new("participant_outcomes").detect { |job| job.display_class == "ParticipantOutcomes::SendToQualifiedTeachersApiJob" }
```

Reply:
That's fair, however I think I'd like to avoid mapping all jobs in Sidekiq (as there might be quiet a few in other queues) to only find the jobs in the `participant_outcomes` queue, and even though it's not ideal in the tests to stub this at least we ensure we are very explicit on exactly what we are checking